### PR TITLE
[B2BORG-114] Autocomplete organization selector

### PR DESCRIPTION
**Pull request by polishq**
This PR updates the "B2B information" component displayed within the `admin-customers` app when editing a user. Previously, we provided simple dropdowns for the user to select a role, organization, and cost center. However, in accounts where there are many many organizations, this does not provide a good experience. This PR instead implements an autocomplete input for the user to select their organization, and then a simple dropdown to select a cost center from within that org. 

Additionally, this PR provides a new "REMOVE B2B INFO" button to completely delete a user's role/org/costcenter. 

Some styling was also applied to the B2B info section to make it more clear that it is separate from the other controls on the page. There is now a "B2B Information" heading and a border around the section.

New version is linked here: https://b2bsuite--sandboxusdev.myvtex.com/admin/customers

image

### CODE REVIEW

File: react/components/OrganizationsAutocomplete.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      onClear()
+    }
+  }, [term])
+
+  const input = {
+    onChange: (_term: string) => {
+      setTerm(_term)
+    },
+    onClear,
+    placeholder: 'Search organization...',
```
Comment (arturmagalhaesjr) [inline react/components/OrganizationsAutocomplete.tsx:94]: you missed the intl here.
I've already done it in the b2b-org PR, you can copy from that.

**Review by arturmagalhaesjr — CHANGES_REQUESTED** [review]

File: react/components/OrganizationsAutocomplete.tsx (hunk 1 above)
Comment (polishq) [reply react/components/OrganizationsAutocomplete.tsx:94]: Haha thanks

Comment (polishq) [reply react/components/OrganizationsAutocomplete.tsx]: Done

**Review by wender — APPROVED** [review]

**Review by arturmagalhaesjr — APPROVED** [review]
LGTM

**Pull request merged**